cunumeric/random/generator.py

```diff
@@ -363,6 +386,31 @@ def zipf(
 
 
 def default_rng(seed: Union[int, None] = None) -> Generator:
+    """
+        Construct a new Generator with the default BitGenerator (XORWOW).
+
+    Parameters
+    ----------
+    seed : {None, int,  BitGenerator, Generator}, optional
+        A seed to initialize the `BitGenerator`. If None, then fresh,
```

Comment:
If this is intended to render as an inline literal, then for ReST it has to be double-backticks, not single (here and elsewhere)

Reply:
3f68ce4aacfbb28323d4ef611e7badaccfed9ac4